run.py

```diff
@@ -1011,6 +1011,14 @@ def main():
     # Sort alphabetically.
     tests.sort()
 
+    # If we don't find any tests, let's let the user know and exit
+    if not tests:
+        if args.testdir:
+            print("Warning: Couldn't find any directory to match the pattern %s" % args.testdir)
+        else:
+            print("Warning: Couldn't find any tests to match the pattern %s" % args.patterns)
```

Comment:
I think we need a for loop here to traverse over all the patterns.

Reply:
Oh, when I tested locally, it worked like that. Do we want to have one line for each pattern?
![image](https://user-images.githubusercontent.com/16976857/155099569-a06a905d-5726-4928-9571-54b3223ee6ef.png)

Reply:
Yeah, I was thinking more like
`Warning: Couldn't find any tests to match the pattern "abc"`
or
`Warning: Couldn't find any tests to match the patterns: "abc", "def"`

you can leave it be if it does not make sense to you. ;)

Reply:
That's not for me to decide!! :P I'll work some more on it, thanks :)